_includes/c11-cpp11-and-beyond-and-toolchains.md

```diff
@@ -134,7 +134,7 @@ matrix:
       env:
         - MATRIX_EVAL="CC=clang-3.9 && CXX=clang++-3.9"
 
-    # works on Trusty
+    # Currently doesn't work on Trusty
```

Comment:
This one seems to work except for the issue with needing to uninstall gcc or upgrading gcc as well as clang only see's gcc's header files first and so it uses them to compile code. (Which is what makes this fail for anything C++ 14 or newer.

Reply:
seems related to https://github.com/travis-ci/travis-ci/issues/9550#issuecomment-433030978
i.e. just need to add CPATH to MATRIX_EVAL should be sufficient IMHO

Reply:
working example: https://github.com/abseil/abseil-cpp/pull/197/files#diff-354f30a63fb0907d4ad57269548329e3